src/nidcpower/examples/nidcpower_advanced_sequence.py

```diff
@@ -29,17 +29,17 @@ def example(resource_name, channels, options, steps, voltage_start, voltage_fina
         session.source_delay = datetime.timedelta(seconds=0.1)
         session.voltage_level_autorange = True
         session.current_level_autorange = True
-        session.create_advanced_sequence(sequence_name='my_sequence', attribute_ids=attribute_ids)
```

Comment:
I don't really want to just remove this example, but it fails the build unless I change it to use these now private functions.